components/sidebar/header/sidebar_header.jsx

```diff
@@ -5,28 +5,43 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 
-import PreferenceStore from 'stores/preference_store.jsx';
-import {Constants, Preferences, TutorialSteps} from 'utils/constants.jsx';
+import {Constants} from 'utils/constants.jsx';
 import * as Utils from 'utils/utils.jsx';
 import StatusDropdown from 'components/status_dropdown/index.jsx';
 import MenuTutorialTip from 'components/tutorial/menu_tutorial_tip';
 
 import SidebarHeaderDropdown from './dropdown';
 
-export default class SidebarHeader extends React.Component {
+export default class SidebarHeader extends React.PureComponent {
+    static propTypes = {
+        teamId: PropTypes.string.isRequired,
+        teamDisplayName: PropTypes.string.isRequired,
+        teamDescription: PropTypes.string.isRequired,
+        teamName: PropTypes.string.isRequired,
+        teamType: PropTypes.string.isRequired,
+        currentUser: PropTypes.object.isRequired,
+        showTutorialTip: PropTypes.bool.isRequired,
+    };
+
+    static defaultProps = {
+        teamDisplayName: '',
```

Comment:
I know it's just being relocated, but should we drop the `isRequired` for these props?

Reply:
Just removed the defaults. That's probably better as they really are required.